rental.rb

```diff
@@ -0,0 +1,17 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] Getters and Setters for `@book` and `@person` as for `@date`, `book and person` should be `attr_accessor` as well. Kindly fix that 🙏🏾

Reply:
✔️ Checked